modules/data/log/tmpl/index.tmpl

```diff
@@ -0,0 +1,40 @@
+<? I18N znc-log ?>
+<? INC Header.tmpl ?>
+
+<? IF Files ?>
+<h3>File Browser</h3>
+<div class="sectionbg">
+<div class="sectionbody">
+<ul style="list-style: none;">
+<? LOOP Files ?>
+<li><a href="<? VAR ModPath TOP ?>?path=<? VAR Long ESC=HTML,URL ?>"><? VAR Short ?></a></li>
+<? ENDLOOP ?>
+</ul>
+</div>
+</div>
+<? ENDIF ?>
+
+<? IF Log ?>
+<h3>Log Viewer (Page <? VAR Page ?>)</h3>
+<div class="sectionbg">
```

Comment:
Could you indent the lines here? It's hard to understand which div is inside which div

Reply:
Ok. I'll redo the indentation here after I have done some other revisions. I just wasn't too sure how I should do it since other templates don't seem to follow a consistent pattern.

Reply:
I redid the indentation while I was redoing the template flow for the Global module. This should be resolved now.